.github/ISSUE_TEMPLATE/1.bug.yml

```
@@ -34,11 +34,11 @@ body:
     attributes:
       label: Steps to reproduce 🕹
       description: |
-        Provide a link to a live example (you can use codesandbox.io) and an unambiguous set of steps to reproduce this bug.
-        Include code to reproduce, if relevant (which it most likely is).
-
-        You can use the official codesandbox template as a starting point: https://mui.com/r/issue-template.
+        Please provide a link to a live example and an unambiguous set of steps to reproduce this bug.
+             
+        As a starting point, we recommend you browse our [documentation](https://mui.com/material-ui/getting-started/installation/), and [select](https://mui.com/static/core/forking-an-example.png) the closest example to your use case.
```

If we want to consider the docs infrastructure with its own namespace, then maybe it would make sense to do this:

```suggestion
        As a starting point, we recommend you browse our [documentation](https://mui.com/material-ui/getting-started/installation/), and [select](https://mui.com/static/docs/forking-an-example.png) the closest example to your use case.
```

But we could also consider that bug reproduction is essential part of the process of Core, than X extends, in which case we are good. I personally lean on keeping core because this logic won't work with MUI Design Kit and Toolpad that both can use our docs infrastructure.

---

I think both work. But revisiting this issue, I think `docs` would be a better name for this folder, even if the asset is not used by all the products. It's also a more intuitive name to find (and keep content organized) next time something similar comes up.